public/static/locales/en/communities.json

```
@@ -0,0 +1,9 @@
+{
+    "allCommunities": "All Communities",
+    "description": "Description",
+    "helpText": "Communities are public groups of apps. Creating a community allows you to create a custom collection of apps, and other CyVerse members can follow that community. When following a community, that community's app listing becomes available in the Apps view.",
```

Might want to rethink this wording a little bit. `Communities are public groups of apps` - Right now communities works only with `Apps`. But it might be expanded to data. Also `When following a community, that community's app listing becomes available in the Apps view` - makes me think that I have to follow a community to access a particular app.

---

If we want a more open description for the future, maybe something like:
```suggestion
    "helpText": "Communities can feature a curated collection of public resources in the Discovery Environment, such as apps. Creating a community allows you to create a custom collection of resources, and other CyVerse members can follow that community. When following a community, that community's custom listing of resources becomes easily accessible, such as its own custom listing in the Apps view.",
```
We probably want this text reviewed by more non-devs 🤔